### ISSUE TITLE
Tree View conflicts with deleting pages

### ISSUE THREAD

**Issue by paletochen** [joekrill/silverbullet-treeview#19]
When having Tree View installed, it seems my instance is not deleting pages. Without it, it works.

Instance in docker, tried in `online mode`

```
Deleting page from space editor.plug.js:1:10463
Deleting page space.ts:60:14
XHRDELETE
https://sb.domain.com/test music.md
[HTTP/2 500  4ms]

An exception was thrown as a result of invoking function deletePage error: Failed to delete file: editor.plug.js:1:696
Error running command Failed to delete file: editor_ui.tsx:146:28
Flushed widget cache to store
```

the XHR Delete has the following Response `Unregistered syscall clientStore.get`

**Comment by joekrill** [comment]
I'm not able to reproduce it in my instance. From the error message, the problem _looks_ to be coming from the editor component. But there _was_ a similar problem with the tree view plug a while back that has since been fixed. 

- Do you get the delete confirmation modal? Or does it fail before even showing that?
- Do you have the latest version of the Tree View plug installed (`Ctrl-/`, `Plugs: Update`)? 
- What version of SilverBullet are you running (`Ctrl-/`, `Help: Version`)? 
- Does it work in sync mode? 
- Does it work when the tree view plug is installed, but the tree view panel itself is closed?

**Comment by paletochen** [comment]
>     * Do you get the delete confirmation modal? Or does it fail before even showing that?

Yes.

>     * Do you have the latest version of the Tree View plug installed (`Ctrl-/`, `Plugs: Update`)?

I guess I had it as i run `PLUGS: Update` regularly

>     * What version of SilverBullet are you running (`Ctrl-/`, `Help: Version`)?

0.8.4

>     * Does it work in sync mode?

I don't recall right now
 
>     * Does it work when the tree view plug is installed, but the tree view panel itself is closed?

No, treeview was not open and it was not working


FYI,
I tried to install the plugin again to make more tests but when installing it, my instance stops behaving.
For example: I cannot open the Page/Template picker or the command selection.

I once tried to add the treeview to the sidebar and when pressing the icon it was throwing an error (I couldn't capture the error). After that, my instance was unresponsive and could not do more tests.

Deleting treeview brings back my instance to a normal state

**Comment by raphwriter** [comment]
> FYI,
I tried to install the plugin again to make more tests but when installing it, my instance stops behaving.
For example: I cannot open the Page/Template picker or the command selection.

I believe this might be related to https://github.com/silverbulletmd/silverbullet/issues/986 @paletochen 
It's fixed in 0.8.5

**Comment by paletochen** [comment]
Thanks for the pointer.
I have 0.8.5. I installed again treeview and can confirm that now it doesn't conflict with deleting pages
